### PR TITLE
Feature/team page

### LINKED ISSUE #180: Team Page
Using structured data to drive the content of the page

### PR DESCRIPTION
**Pull request by JacOng17**
- Created a Getting Started doc.
- Configured Team page.

closes #180

### CODE REVIEW

File: _sass/_components/layout.scss [hunk 1]
```diff
@@ -129,6 +129,9 @@
 
 .background-gray {
   background-color: $color-gray-lightest;
+  h2 {
```
[the remark this answers is not in the document]
Comment (Br3nda) [reply _sass/_components/layout.scss]:
```suggestion

   h2 { 
```

File: _sass/_components/layout.scss [hunk 2]
```diff
@@ -129,6 +129,10 @@
 
 .background-gray {
   background-color: $color-gray-lightest;
+
```
[the remark this answers is not in the document]
Comment (Br3nda) [reply _sass/_components/layout.scss]:
```suggestion
```

**Comment by Br3nda** [comment]
Needs to be in a menu somewhere. @gthurstonnz  needs to decide where/how to add it.

File: _authors/matti-schneider.md [hunk 3]
```diff
@@ -4,5 +4,5 @@ first_name: Matti
 last_name: Schneider
 full_name: Matti Schneider
 published: true
+status: nil
```
Comment (Br3nda) [inline _authors/matti-schneider.md]: how about:
```suggestion
active: false
```

Comment (JacOng17) [reply _authors/matti-schneider.md]: Looks good. Will change it for all :smile:

**Comment by Br3nda** [comment]
@JacOng17 @gthurstonnz Let's get the real team members on here before we merge it.
Jacob can you remove the fake data, and write up instructions on how Glen can add himself?

File: tests/schema/_authors.yml [hunk 4]
```diff
@@ -1,8 +1,9 @@
 ---
 name: String
 first_name: String
+last_name: String
 full_name: String
-published: Boolean
+status: String
```
Comment (Br3nda) [inline tests/schema/_authors.yml]: boolean here

File: _plugins/update_author_data.rb [hunk 5]
```diff
@@ -4,14 +4,15 @@
 # Detects if authors have published posts
 # And renders a page for them accordingly
 # at /author/:name/
-Jekyll::Hooks.register :site, :after_init do
```
Comment (Br3nda) [inline _plugins/update_author_data.rb]: If we're not using this plugin any more, let's remove it.

(as long as we understand what it was doing)

**Comment by gthurstonnz** [comment]
> Needs to be in a menu somewhere. @gthurstonnz needs to decide where/how to add it.

@JacOng17 This would preferrably need to be under the about page after history

**Comment by JacOng17** [comment]
Just need to confirm with Glen where exactly the Team page appears b4 merging :smile: 

Will check-in with him when I'm running him through the instructions later.

**Pull request merged**